romancal/lib/tests/helpers.py

```diff
@@ -0,0 +1,42 @@
+import os
+import glob
+import io
+
+
+S3_BUCKET_NAME = "test-s3-data"
+
+
+class MockS3Client:
```

Comment:
This whole module is a good candidate for `stcal`.

Reply:
@jdavies-st Is `stcal` or `ci_watson` a better place for it?

Reply:
There's already an s3 client library that Ed wrote that both `jwst` and `crds` use:

https://github.com/spacetelescope/stsci-aws-utils

Maybe the mocker could go there?  @eslavich ?